xml/Microsoft.VisualBasic.ApplicationServices/WindowsFormsApplicationBase.xml

```diff
@@ -1083,7 +1083,7 @@
 ||  
 |-|  
 |To access the Code Editor window for application events|  
-|1.  With a project selected in **Solution Explorer**, click **Properties** on the **Project** menu.<br />2.  Click the **Application** tab.<br />3.  Click the **View Application Events** button to open the Code Editor.<br /><br /> For more information, see [Application Page, Project Designer (Visual Basic)](../Topic/Application%20Page,%20Project%20Designer%20\(Visual%20Basic\).md).|  
+|1.  With a project selected in **Solution Explorer**, click **Properties** on the **Project** menu.<br />2.  Click the **Application** tab.<br />3.  Click the **View Application Events** button to open the Code Editor.<br /><br /> For more information, see [Application Page, Project Designer (Visual Basic)](http://msdn.microsoft.com/library/8cec9fea-cd92-47ff-88dd-7c928f0b4a74).|  
```

Comment:
The topic exists in docs: https://docs.microsoft.com/en-us/visualstudio/ide/reference/application-page-project-designer-visual-basic